lib/group/group-router.js

```diff
@@ -1,56 +1,66 @@
 'use strict';
 
 var express = require('express'),
-  config = require('./config-group');
+  config = require('./config-group'),
+  shortid = require('shortid');
 
-function initRouter(mediator) {
+var GroupCloudDataTopics = require('fh-wfm-mediator/lib/topics');
+
+/**
+ * Function which handles the response according to the promise returned by the topic
+ *
+ * @param {Object} res
+ * @param {Promise} topicPromise Promise returned by the topic the request was sent to
+ */
+function handleResponse(res, topicPromise) {
+  topicPromise.then(function(data) {
+    res.status(200).json(data);
+  }).catch(function(err) {
+    console.log("[Error] Group topic request: ", err);
+    res.status(500).json({error: err.message});
+  });
+}
```

Comment:
Ah, yes, just tested this and they do swallow all the exceptions. I guess the way to go then is
```javascript
(req, res, next) => {
// ...
  .catch(next)
}
```

There's no difference between 2-arity and 3-arity handlers btw, you can still just not call `next` if you're ending the response. Only 4-arity is special-cased for error handling.